TicTacToeAppTests/ViewControllerSpec.swift

```diff
@@ -0,0 +1,101 @@
+import Quick
+import Nimble
+import TicTacToeApp
+
+class ViewControllerSpec: QuickSpec {
+  override func spec() {
+
+    describe("ViewController") {
+
+      it("disables button after making a move") {
+        let controller = ViewController()
+        let button = UIButton()
+
+        controller.makeMove(button)
+
+        expect(button.enabled).to(beFalse())
+      }
+
+      it("changes button title after making a move") {
+        let controller = ViewController()
+        let button = UIButton()
+
+        controller.makeMove(button)
+
+        expect(button.currentTitle).notTo(beNil())
+      }
+
+      it("changes button title to alternating player marks after each move") {
```

Comment:
This looks like it should have been multiple tests. Remember the three rules of TDD, specifically the one about "writing no more of a unit test than is sufficient to fail."